cupyx/scipy/sparse/linalg/_eigen.py

```diff
@@ -0,0 +1,158 @@
+import numpy
+import cupy
+
+
+def eigsh(a, k=6, which='LM', ncv=None, maxiter=None, tol=0,
+          return_eigenvectors=True):
```

Comment:
Would you make the parameters following `k` keyword-only ones in case additional SciPy compatible parameters are supported in the future?

Reply:
Just to confirm, you are suggesting to use `kwargs` for keyword arguments execept `k`, correct?

Reply:
No, I suppose the following should work. (And `k` can be positional as it follows `a` just after in SciPy)
```suggestion
def eigsh(a, k=6, *, which='LM', ncv=None, maxiter=None, tol=0,
          return_eigenvectors=True):
```